apps/nextjs/src/components/AppComponents/QuizDesigner/Hero.tsx

```diff
@@ -129,9 +129,14 @@ const Hero = ({
                     trackEvent("quiz_designer:begin");
                     dispatch({ type: QuizAppActions.Begin });
                   }
-                  questionsWrapperRef.current?.scrollIntoView({
-                    behavior: "smooth",
-                  });
+                  const questionsWrapper = questionsWrapperRef.current;
+                  if (questionsWrapper) {
+                    const { top } = questionsWrapper.getBoundingClientRect();
+                    window.scrollTo({
+                      top: top + window.pageYOffset - 200,
+                      behavior: "smooth",
```

Comment:
I think we can add `<html class="scroll-smooth">` so we don't need to re-specify smooth scrolling everywhere